test_main.py

```
@@ -23,7 +25,9 @@ def test_scrape_website_success(mock_get, scraper):
 @patch("main.requests.Session.get")
 def test_scrape_website_http_error(mock_get, scraper):
     mock_response = MagicMock()
-    mock_response.raise_for_status.side_effect = requests.exceptions.HTTPError("404 Not Found")
+    mock_response.raise_for_status.side_effect = requests.exceptions.HTTPError(
+        "404 Not Found"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)